mantle/cmd/kola/qemuexec.go

```diff
@@ -113,6 +116,25 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 		}
 		config = &configv
 	}
+	if fcct != "" {
+		if config == nil {
+			config = &v3types.Config{}
+		}
+		if directIgnition {
+			return fmt.Errorf("Cannot use --fcct with --direct-ignition")
+		}
+		f, err := os.Open(fcct)
```

Comment:
Minor: note `fcct` can also just take the file to read as an argument. Would simplify the logic here a bit.